doc/man/restic-dump.1

```diff
@@ -42,6 +42,10 @@ Exit status is 12 if the password is incorrect.
 \fB-a\fP, \fB--archive\fP="tar"
 	set archive \fBformat\fR as "tar" or "zip"
 
+.PP
+\fB--compress\fP[=false]
```

Comment:
The man pages are only regenerated directly before a release. Please revert that change.